mlrun/db/httpdb.py

```diff
@@ -3896,7 +3896,7 @@ def patch_model_endpoint(
         function_name: Optional[str] = None,
         function_tag: Optional[str] = None,
         endpoint_id: Optional[str] = None,
-    ) -> mlrun.common.schemas.ModelEndpoint:
+    ) -> None:
```

Comment:
remove the 
`:return:                          The updated `ModelEndpoint` object.`
from the func docs